### ISSUE TITLE
Replace Kafka base image

### ISSUE THREAD

**Issue by ewilkins-csi** [boozallen/aissemble#51]
## Background
The base image we are using for the `aissemble-kafka` image has been removed from Docker Hub.  We need a suitable replacement. Because our `aissemble-kafka-chart` helm chart is based on Bitnami's chart, the Bitnami kafka image is likely the most suitable replacement.

## Definition of Done
 * Replace wurstmeister/kafka with bitnami/kafka
 * Update the docker build script for aissemble-kafka to preserve customization in the new image
 * Update the kafka helm chart to use the aissemble-kafka image

## Test Script
1. Create a new project
```
mvn archetype:generate -B -DarchetypeGroupId=com.boozallen.aissemble \
                          -DarchetypeArtifactId=foundation-archetype \
                          -DarchetypeVersion=1.7.0-SNAPSHOT \
                          -DartifactId=test-project-51\
                          -DgroupId=org.test \
                          -DprojectName='Test Kafka Image' \
                          -DprojectGitUrl=test.org/test-project-51\
&& cd test-project-51
```
2. Add [TestPipeline.json](https://github.com/boozallen/aissemble/files/15251362/TestPipeline.json) to _test-project-51-pipeline-models/src/main/resources/pipelines_
3. Install the model file
```
mvn clean install
```
4. Follow any manual actions and re-run MDA generation. Repeat until no more actions are printed.
```
mvn clean generate-sources
```
5. Update the `executeStepImpl` method of _test-project-51-pipelines/test-pipeline/src/main/java/org/test/Ingest.java_ to copy received messages to the outbound topic
```
	protected String executeStepImpl(String inbound) {
		logger.info("Received message:" + inbound);
		return "Passing message: " + inbound;
	}
```
6. Build the project
```
mvn clean install
```
7. Deploy the project and wait for all resources to start.
```
tilt up; tilt down
```
8. Execute the _test-pipeline_ resource in the Tilt UI and wait for it to start. (The final messages once the pipeline is fully started are about the kafka consumer subscription being created.)
9. Attach to the kafka pod and start a console consumer on topic _output_
```
kubectl exec -it kafka-cluster-0 -- /bin/bash
kafka-console-consumer.sh --bootstrap-server localhost:9093 --topic output
```
10. Attach a new console to the kafka pod and start a console producer on topic _ingest_
```
kubectl exec -it kafka-cluster-0 -- /bin/bash
kafka-console-producer.sh --bootstrap-server localhost:9093 --topic ingest
```
11. Type any message into the producer console and hit enter
12. Verify the message is passed along to the consumer created in step 9, prepended with _"Passing message: "_. (This might take around 15-20 seconds the first time.)

**NOTE:** For OTS it will be necessary to build the entrie `aissemble-parent` project and the `mda-maven-plugin`,`aissemble-kafka` and `aissemble-kafka-chart` modules with `--also-make` in the baseline. It's also necessary to edit the _test-project-51-deploy/src/main/resources/apps/kafka-cluster/Chart.yaml_ to point to the aissemble-kafka-chart on the local filesystem.  Finally, disable baton so it does not revert the changes to Chart.yaml.

**Comment by ewilkins-csi** [comment]
OTS with @Cho-William

**Comment by ewilkins-csi** [comment]
**Merged** in #53

**Comment by nartieri** [comment]
Successfully verified.
